README.md

```diff
@@ -1,2 +1,55 @@
 # dashboard
 Web UI for KubeEdge
+
+## Environment Prepare
+
+Install `node_modules`:
+
+```bash
+npm install
+```
+
+or
+
+```bash
+yarn
```

Comment:
Please correct the command execute order: first `yarn` and then `npm run start:dev`